lambda/recipes-responder/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+//todo - as of now creating for recipes-responder but we can check if we need constant file globally means at root of the project to hold all constants?)
+
+export const INDEX_JSON = "index.json"
+export const V2_INDEX_JSON = "v2/index.json"
```

Comment:
These need to go into the common library as they are currently breaking module scoping rules by being imported in writeIndexData which lives in the common library

Reply:
Thanks @fredex42 . constants file moved to common folder like writeIndex as suggested and hence solves todo question too to keep constants file here or somewhere else.